### DISCUSSION TITLE
ValueError: Input signal length=0 is too small to resample from 8000->16000

### DISCUSSION THREAD

**Discussion by JakobH3** [NVIDIA-NeMo/NeMo#5531] [General]
When attempting to train a quartznet model on our data we get the error below. The trainer is able to initialize the model but it errors out right as the first epoch begins. 

![image](https://user-images.githubusercontent.com/60677704/204891077-c4373f0b-b56e-422a-a5aa-ffb5d8acb5e6.png)

![image](https://user-images.githubusercontent.com/60677704/204889956-dc1312bd-cee3-4356-abac-37595ae4b9ad.png)

This is the code we are using to train the model

import nemo.collections.asr as nemo_asr
import pytorch_lightning as pl
import torch
import gc
from ruamel.yaml import YAML
from omegaconf import DictConfig

config_path = 'config/config.yaml'
train_manifest = 'utils/manifests/atcc_train.json'
valid_manifest = 'utils/manifests/atcc_validation.json'
checkpoint_name = '100epoch_quartznet_scratch.nemo'

torch.cuda.empty_cache()
gc.collect()

with open(config_path, 'r') as f:
    config = YAML(typ='safe').load(f)

config['model']['train_ds']['manifest_filepath'] = train_manifest
config['model']['train_ds']['batch_size'] = 8
config['model']['validation_ds']['manifest_filepath'] = valid_manifest
config['model']['validation_ds']['batch_size'] = 8

model = nemo_asr.models.EncDecCTCModel(cfg=DictConfig(config['model']))
model.setup_training_data(DictConfig(config['model']['train_ds']))
model.setup_validation_data(DictConfig(config['model']['validation_ds']))

trainer = pl.Trainer(gpus=[0], max_epochs=100)
trainer.fit(model)

model.save_to(checkpoint_name)

We are using librosa version 0.9.2.

**Comment by titu1994** [comment]
It is possible that the audio file is corrupt (0 length audio). The errors occurs the moment the dataloader tries to read the audio file, so please check for any corrupt audio files.
